.github/workflows/dashboard_deploy_template.yaml

```diff
@@ -0,0 +1,80 @@
+name: Dashboard Deploy
+
+on:
+  workflow_call:
+    inputs:
+      parameter_file:
+        description: "Path to the parameters file"
+        required: true
+        type: string
+
+jobs:
+  deployShiny:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install yq
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y yq
+
+      - name: Read parameters
+        run: |
+          DASHBOARD_NAME=$(yq '.dashboard_name' ${{ inputs.parameter_file }})
+          DEPLOY_TARGET=$(yq '.deploy_target' ${{ inputs.parameter_file }})
+          INSTALL_GDAL=$(yq '.install_gdal' ${{ inputs.parameter_file }})
+          EXTRA_PACKAGES=$(yq '.extra_packages | join(" ")' ${{ inputs.parameter_file }})
+
+          echo "DASHBOARD_NAME=$DASHBOARD_NAME" >> $GITHUB_ENV
+          echo "DEPLOY_TARGET=$DEPLOY_TARGET" >> $GITHUB_ENV
+          echo "INSTALL_GDAL=$INSTALL_GDAL" >> $GITHUB_ENV
+          echo "EXTRA_PACKAGES=$EXTRA_PACKAGES" >> $GITHUB_ENV
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          use-public-rspm: true
+
+      - name: Install GDAL
+        if: env.INSTALL_GDAL == 'true'
```

Comment:
If we do change the name to "Install Terra dependencies", then I'd call this variable INSTALL_TERRA_DEPENDENCIES instead, I think that would be clearer? (Wary that some analysts using this and trying to set their yml file might never have heard of GDAL or realise what it is)